Rubrik/Public/New-RubrikMount.ps1

```diff
@@ -42,6 +42,8 @@ function New-RubrikMount
     [Parameter(Position = 2,ValueFromPipelineByPropertyName = $true)]
     [ValidateNotNullorEmpty()]
     [String]$Date,
+    # SnapshotID of the VM instead of searching for one by date
+    [String]$SnapshotID,
```

Comment:
I like this line of thought, but perhaps instead we use a pipe structure of Get-RubrikSnapshot | New-RubrikMount and make sure that the ValueFromPipelineByPropertyName matches?